### PR TITLE
regular updates and fixes (late August) (2)

### PR DESCRIPTION
**Pull request by jorgeorpinel**

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
I think I'm done here. Ready to merge

**Comment by jorgeorpinel** [comment]
Actually, I'm going to add as many changes as I can to address #448 in this PR while the review can be completed. See for example my latest commit ^

File: static/docs/commands-reference/get.md [hunk 1]
```diff
@@ -37,6 +37,13 @@ created in the current working directory, with its original file name.
 
 ## Options
 
+- `-o`, `--out` - specify a path (directory and file name) to the desired
+  location to place the imported data in. The default value (when this option
+  isn't used) is the current working directory (`.`) and original file name.
+
+- `--rev` - specific Git revision of the DVC repository to import the data from.
+  `HEAD` by default.
```
Comment (shcheklein) [inline static/docs/commands-reference/get.md]: can we provide a link, or mention that it's usually a master's branch last commit? cc @efiop is HEAD technically correct here? just to double check

File: static/docs/changelog/0.35.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ improvements) we have done in the last few months:
 - 📖 The [Get Started](/doc/get-started/agenda) section has been simplified
   (e.g. to use tags instead of branches) and extended. We have also prepared a
   [Github DVC project ](https://github.com/iterative/example-get-started)that
```
Comment (shcheklein) [inline static/docs/changelog/0.35.md]: minor: add a space after )

**Comment by shcheklein** [comment]
@jorgeorpinel please, check one of the comments re the HEAD

**Comment by jorgeorpinel** [comment]
Thanks for merging @shcheklein. I'll address the comments and add a new "regular updates" PR.

File: static/docs/commands-reference/get.md (hunk 1 above)
Comment (jorgeorpinel) [reply static/docs/commands-reference/get.md]: Linked and clarified in c38e6ce.
Good question for @efiop though, I moved it to https://github.com/iterative/dvc.org/pull/601#pullrequestreview-282331391.
p.s. I just copied this from the existing `dvc import` ref.

File: static/docs/changelog/0.35.md (hunk 2 above)
Comment (jorgeorpinel) [reply static/docs/changelog/0.35.md]: This was already fixed. But I removed the extra space before `]` in 07fb5b0.

**Pull request merged**